central/networkgraph/flow/datastore/internal/store/postgres/tests/store_test.go

```diff
@@ -75,33 +75,37 @@ func (s *NetworkflowStoreSuite) TestStore() {
 	secondCluster := fixtureconsts.Cluster2
 	store2 := postgresFlowStore.New(s.pgDB.DB, secondCluster)
 
+	now := time.Now().UTC()
+	timeNow := timestamp.FromGoTime(now)
+
+	time1 := timeNow.Add(time.Second)
+
 	networkFlow := &storage.NetworkFlow{
 		Props: &storage.NetworkFlowProperties{
 			SrcEntity:  &storage.NetworkEntityInfo{Type: storage.NetworkEntityInfo_DEPLOYMENT, Id: "a"},
 			DstEntity:  &storage.NetworkEntityInfo{Type: storage.NetworkEntityInfo_DEPLOYMENT, Id: "b"},
 			DstPort:    1,
 			L4Protocol: storage.L4Protocol_L4_PROTOCOL_TCP,
 		},
-		LastSeenTimestamp: protocompat.GetProtoTimestampFromSeconds(1),
+		LastSeenTimestamp: time1.Protobuf(),
 		ClusterId:         clusterID,
 	}
-	zeroTs := timestamp.MicroTS(0)
 
 	foundNetworkFlows, _, err := s.flowStore.GetAllFlows(s.ctx, nil)
 	s.NoError(err)
 	s.Len(foundNetworkFlows, 0)
 
 	// Adding the same thing twice to ensure that we only retrieve 1 based on serial Flow_Id implementation
-	s.NoError(s.flowStore.UpsertFlows(s.ctx, []*storage.NetworkFlow{networkFlow}, zeroTs))
-	networkFlow.LastSeenTimestamp = protocompat.GetProtoTimestampFromSeconds(2)
-	s.NoError(s.flowStore.UpsertFlows(s.ctx, []*storage.NetworkFlow{networkFlow}, zeroTs))
+	s.NoError(s.flowStore.UpsertFlows(s.ctx, []*storage.NetworkFlow{networkFlow}, timeNow))
+	networkFlow.LastSeenTimestamp = timeNow.Add(2 * time.Second).Protobuf()
+	s.NoError(s.flowStore.UpsertFlows(s.ctx, []*storage.NetworkFlow{networkFlow}, timeNow))
```

Comment:
Why the change from `zeroTs` to `timeNow`?

Reply:
I wasn't able to get this unit test to pass locally without making this change.